test/unit/spec/util/support.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const assert = require('assert');
+const isSupported = require('../../../../lib/util/support');
+
+describe.only('isSupported', () => {
```

Comment:
```suggestion
describe('isSupported', () => {
```

Reply:
thanks for catching this. Maybe I should add an eslint rule that prevents such mistakes.